mkdocs.yml

```diff
@@ -1,11 +1,9 @@
 #
-# Copyright (c) 2018
-# Mainflux
-#
+# Copyright (c) Mainflux
 # SPDX-License-Identifier: Apache-2.0
 #
 
-copyright: Copyright (c) 2015-2019 Mainflux
```

Comment:
I think that you can leave copyright years here, but I don't mind removing them, as well.

Reply:
I actually debated about this one.  My reasoning was that it was more consistent not to have any copyright dates anywhere.